client/components/drop-zone/test/index.jsx

```diff
@@ -37,7 +24,9 @@ describe( 'DropZone', function() {
 	} );
 
 	after( function() {
-		delete window.MutationObserver;
+		if ( global.window && global.window.MutationObserver ) {
```

Comment:
Why did we have to do this in the first place? cleanup lets go of window, which holds MutationObserver...